sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests/DistroWebAppLiveTests.cs

```diff
@@ -73,6 +85,7 @@ public async Task VerifyDistro()
                 .ConfigureResource(x => x.AddAttributes(resourceAttributes))
                 .UseAzureMonitor(options =>
                 {
+                    options.EnableLiveMetrics = false;
```

Comment:
Does this makes a difference? Anyways live metrics are sent to different endpoint.

Reply:
It removes noise when running locally. And the Recorded test captures all network traffic.